wiki/Active-Effect-Guide.md

```diff
@@ -0,0 +1,440 @@
+# FoundryVTT dnd5e Active Effects Examples
+
+![Up to date as of 2.1.x](https://img.shields.io/badge/dnd5e-v2.1.x-informational)
+
+See [Kandashi's Active Effects Guide](https://docs.google.com/document/d/1DuZaIFVq0YulDOvpahrfhZ6dK7LuclIRlGOtT0BIYEo) for a trove of useful information
+
+This document only covers Active Effects available to the Core dnd5e System.
+
+## Legend
+
+`[number]` - These square brakets mean "replace this with your value of the type within the brackets".
+So this example: `+[number]` would mean you input `+3`.
+
+`[formula]` - When `formula` is mentioned in this document it means this value can be populated with any dice formula. For example, Bless adds several effects with the Effect Value of `1d4`.
+
+
+As part of this, an Actor's Rolldata is available as ["@attributes."](https://github.com/foundryvtt/dnd5e/wiki/Roll-Formulas) Useful examples:
```

Comment:
Pretty sure we do not support roll data as Active Effect values.

Reply:
In fields that are strings, we absolutely do. It varies from attribute to attribute. For example, `system.bonuses.<type>.attack`, (a string) but *not* `system.attributes.ac.bonus` (an integer).

Reply:
Hmm, I think we have to invoke a separate roll data pass manually on a case-by-case basis. It's not a general thing supported by the Active Effects system.